analysis/R/association.R

```diff
@@ -353,10 +352,14 @@ ComputeDistributionEM <- function(reports, report_cohorts, maps,
     f <- var_params$f
     pstar <- (1 - f / 2) * p + (f / 2) * q
     qstar <- (1 - f / 2) * q + (f / 2) * p
+    k <- var_params$k
 
-    if (ignore_other) {
+    # Ignore other probability if either ignore_other is set or k == 1
```

Comment:
Why not have the caller set ignore_other if k == 1, instead of putting it in this function?